slack/web/base_client.py

```diff
@@ -273,23 +279,38 @@ async def _request(self, *, http_verb, api_url, req_args):
             await session.close()
         return response
 
-    @staticmethod
-    def _get_user_agent():
-        """Construct the user-agent header with the package info,
-        Python version and OS version.
+    def _sync_send(self, api_url, req_args):
+        params = req_args["params"] if "params" in req_args else None
+        data = req_args["data"] if "data" in req_args else None
+        files = req_args["files"] if "files" in req_args else None
+        _json = req_args["json"] if "files" in req_args else None
```

Comment:
```suggestion
        _json = req_args["json"] if "json" in req_args else None
```

Reply:
oops, good catch.

Reply:
corrected in https://github.com/slackapi/python-slackclient/commit/ece7a12af8c36b571b131d656c418039414542b1